crowbar_framework/lib/tasks/chef.rake

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2011-2013, Dell
+# Copyright 2013-2016, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+namespace :chef do
+  namespace :upload do
+    require "crowbar/chef/upload"
+
+    desc "Upload the complete chef data to the server"
+    task :all do
```

Comment:
Do we need some model or environment access later on? As you are anyway using `Rails` already I would suggest to use `task all: [:environment] do`, than you can drop the require and we've got full access to the rails environment. Should be done for the other tasks as well.